solr_conf/schema.xml

```diff
@@ -631,6 +631,7 @@ v         currently supported on types that are sorted internally as strings
    <field name="sourceResource_type_id" type="string" indexed="true" stored="true" multiValued="true" />
    <field name="sourceResource_type_type" type="string" indexed="false" stored="false" multiValued="true" />
    <field name="sourceResource_type_providedLabel" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="sourceResource_type_name" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
Conversation on this follows from: https://dpla.slack.com/archives/tech/p1430232893000121

Specifically, there's some question to resolve about how we use `prefLabel` and `providedLabel`, and how they should be indexed in this case.